wiki/Static-Analysis-Checks.md

```diff
@@ -227,3 +228,11 @@ To fix the failures for this check: resolve the TODO items and then close the is
 
 # How to add a new Static analysis check
 // TODO([#3690](https://github.com/oppia/oppia-android/issues/3690)): Complete static checks Wiki
+
+# How to run static checks locally
+
+- Go to failing CI check in your GitHub PR.
+- Scroll failing CI check logs to the top and find the Bazel command that was run for this script.
```

Comment:
Scroll to the top of the failing CI check logs, and find the ...

Reply:
Done.